src/hotspot/share/gc/shenandoah/shenandoahPadding.hpp

```diff
@@ -25,6 +25,8 @@
 #ifndef SHARE_GC_SHENANDOAH_SHENANDOAHPADDING_HPP
 #define SHARE_GC_SHENANDOAH_SHENANDOAHPADDING_HPP
 
+#include "memory/padded.hpp"
```

Comment:
This one surprised me, but I'm guessing it was included indirectly
from synchronizer.hpp via some frame header...